src/main/java/hudson/plugins/promoted_builds/inheritance/JobPropertyImplSelector.java

```diff
@@ -0,0 +1,66 @@
+
+package hudson.plugins.promoted_builds.inheritance;
+
+import org.apache.log4j.Logger;
+
+import hudson.Extension;
+import hudson.model.JobProperty;
+
+import hudson.plugins.project_inheritance.projects.InheritanceProject;
+import hudson.plugins.project_inheritance.projects.inheritance.InheritanceSelector;
+
+import hudson.plugins.promoted_builds.JobPropertyImpl;
+
+/**
+ *  
+ * @author Jacek Tomaka
```

Comment:
A good practice is to add <code>@since TODO</code> to the Javadoc. Then it will be possible to resolve these TODOs before the relelase